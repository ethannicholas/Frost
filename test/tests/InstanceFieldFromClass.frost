class InstanceField {
    def x:Int

    @class
    method test() {
        Console.printLine(x)
    }
}

method main() {
    Console.printLine(InstanceField.x)
}