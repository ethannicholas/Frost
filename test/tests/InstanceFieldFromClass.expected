InstanceFieldFromClass.frost:6:27: error: cannot reference 'self' from a @class context
InstanceFieldFromClass.frost:11:23: error: instance field 'x' cannot be accessed from a class context
2 errors
